.gitlab/pipelines/stages/deploy/deploy_kubecf.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+# shellcheck disable=SC1090
+source "$(bazel info workspace)/.gitlab/pipelines/config/config.sh"
+
+# Locate the built kubecf chart.
+chart="$(find output/ -name 'scf-*.tgz')"
```

Comment:
Hmm. Not sure I like finding _any_ file with vaguely the right name.  But then I don't like my alternative either:

```bash
bazel aquery //deploy/helm/scf:chart 2>&1 | grep 'Outputs:' | cut -d: -f2- | y2j | jq -r '.[0]'
```

(Also, you might not have `y2j` and `jq` installed)

Reply:
We don't have `y2j` and `jq` available, but we can achieve the same with `awk`, which we do have:
```
bazel aquery //deploy/helm/scf:chart 2> /dev/null | awk 'match($0, /Outputs: \[(.*)\]/, output){ print output[1] }'
```